examples/http-timeout/http_timeout.bal

```diff
@@ -1,46 +1,28 @@
 import ballerina/http;
-import ballerina/log;
 import ballerina/lang.runtime;
 
-http:Client backendClientEP = check new ("http://localhost:8080", {
-        // Timeout configuration.
-        timeout: 10
-
-    });
+final http:Client backendClientEP = check new ("http://localhost:8080", {
+    // Timeout configuration.
+    timeout: 10
+});
 
 // Create an HTTP service bound to the listener endpoint.
 service /timeout on new http:Listener(9090) {
 
-    resource function get .(http:Caller caller, http:Request request) {
-        http:Response|error backendResponse =
-            backendClientEP->forward("/hello", request);
+    resource function get .() returns http:InternalServerError|string|error? {
+        string|error backendResponse =
+                        backendClientEP->get("/hello");
 
-        // If `backendResponse` is an `http:Response`, it is sent back to the
+        // If `backendResponse` is an string (text/plain), it is sent back to the
         // client. If `backendResponse` is an `http:ClientError`, an internal
         // server error is returned to the client.
-        if (backendResponse is http:Response) {
-            var responseToCaller = caller->respond(<@untainted>backendResponse);
-            if (responseToCaller is error) {
-                log:printError("Error sending response",
-                                'error = responseToCaller);
-            }
+        if (backendResponse is string) {
+            return backendResponse;
         } else {
-            http:Response response = new;
-            response.statusCode = http:STATUS_INTERNAL_SERVER_ERROR;
-            string errorMessage = backendResponse.message();
-            string expectedMessage = "Idle timeout triggered before " +
-                "initiating inbound response";
-            if (errorMessage == expectedMessage) {
-                response.setPayload(
-                    "Request timed out. Please try again in sometime."
-                );
+            if (backendResponse is http:IdleTimeoutError) {
+                return { body: "Request timed out. Please try again in sometime."};
```

Comment:
This line seems to be exceeding the column length. 
```
[/examples/http-timeout/http_timeout.bal] Line number 23.
```

Reply:
Fixed it.